src/nccl_ofi_sendrecv.c

```diff
@@ -960,7 +960,7 @@ static int flush(nccl_net_ofi_recv_comm_t *recv_comm, int n, void **buffers,
 			CU_FLUSH_GPU_DIRECT_RDMA_WRITES_TO_OWNER);
 
 		if (cuda_ret != CUDA_SUCCESS) {
-			ret = ncclUnhandledCudaError;
+			ret = -EPERM;
```

Comment:
Do we need to map EPERM->ncclUnhandledCudaError in nccl_net_ofi_retval_translate?

Reply:
I frankly don't think it matters.  I'm not sure it ever made sense to return this error code.